class.jetpack-cli.php

```diff
@@ -715,7 +715,8 @@ public function sync_queue( $args, $assoc_args ) {
 				$items = $queue->peek( 100 );
 
 				if ( empty( $items ) ) {
-					WP_CLI::log( sprintf( __( 'Nothing is in the %s queue', 'jetpack' ), $queue_name  ) );
+					/* translators: 1: Name of the queue, either 'incremental' or 'full' */
```

Comment:
Almost :) `/* translators: %s is the name of a queue, either 'incremental' or 'full' */`